qkeras/quantizers.py

```diff
@@ -431,10 +431,10 @@ def _clip_po2_scale(scale: tf.Tensor, min_po2_exponent: Any,
   return scale
 
 
-def _get_scale(alpha: Any, x: tf.Tensor, q: tf.Tensor,
-               scale_axis: Any = None, per_channel_scale: bool = True,
-               elements_per_scale: Any = None, min_po2_exponent: Any = None,
-               max_po2_exponent: Any = None):
+def _get_least_squares_scale(
```

Comment:
For backward compatibility purpose, consider to define _get_scale by calling _get_least_squares_scale or vice versa.

Reply:
Good thought. Done